osu.Game.Rulesets.Osu/UI/Cursor/OsuCursorContainer.cs

```diff
@@ -88,6 +91,14 @@ protected override void LoadComplete()
             calculateScale();
         }
 
+		protected override void Update() {
+			foreach (IHidesCursorTrail modifier in Mods.OfType<IHidesCursorTrail>())
+            {
+                cursorTrail.Alpha = 0;
+            } // FIXME: Don't do this every frame. It's only like this right now because for some reason it won't work if I only do it once. 
+			base.Update();
+		}
+
```

Comment:
If you want to get the PR into a mergeable state this definitely needs to be fixed

Reply:
Will fix!

Reply:
Fixed!